src/dev/flang/tools/docs/Html.java

```diff
@@ -387,16 +412,16 @@ private String redefines0(AbstractFeature af)
   private String mainSection(Map<AbstractFeature.Kind, TreeSet<AbstractFeature>> map, AbstractFeature outer)
   {
     // Type Parameters
-    var typeParameters = new TreeSet<AbstractFeature>();
+    var typeParameters = new LinkedList<AbstractFeature>();
     typeParameters.addAll(map.getOrDefault(AbstractFeature.Kind.TypeParameter, new TreeSet<AbstractFeature>()));
     typeParameters.addAll(map.getOrDefault(AbstractFeature.Kind.OpenTypeParameter, new TreeSet<AbstractFeature>()));
     typeParameters.addAll(outer.typeArguments());
 
     // Fields
-    TreeSet<AbstractFeature> fields =  new TreeSet<AbstractFeature>();
+    var fields =  new LinkedList<AbstractFeature>();
     fields.addAll(map.getOrDefault(AbstractFeature.Kind.Field, new TreeSet<AbstractFeature>()));
     var normalArguments = outer.arguments().clone();
-    normalArguments.removeIf(a->a.isTypeParameter() || a.visibility().eraseTypeVisibility() != Visi.PUB);
+    normalArguments.removeIf(a->a.isTypeParameter());// || a.visibility().eraseTypeVisibility() != Visi.PUB);
```

Comment:
comment missing, why code was comented

Reply:
good that you saw that, it shouldn't be commented out, was for debugging only